### PR TITLE
Fix overflow of latex in Callouts

### PR DESCRIPTION
**Pull request by LuciferUchiha**
As mentioned here: https://github.com/shuding/nextra/issues/1516

### CODE REVIEW

**Review by B2o5T — APPROVED** [review]
Could you add a changeset, so users will know about this fix?

**Comment by LuciferUchiha** [comment]
@B2o5T How? I can't see any guide in the README or contrib

**Comment by B2o5T** [comment]
`pnpm changeset` and follow steps

File: .changeset/shy-rockets-run.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'nextra-theme-docs': major
```
Comment (B2o5T) [inline .changeset/shy-rockets-run.md:2]:
```suggestion
'nextra-theme-docs': patch
```

**Pull request merged**